elliott/elliottlib/cli/find_builds_cli.py

```
@@ -733,15 +736,98 @@ def _filter_out_attached_builds(
+    Arg(s):
+        delivery_repo (str): Delivery repository name (e.g. "openshift4/ose-cli-rhel9").
+        version (str): Build version (e.g. "4.19.0").
+        release (str): Build release (e.g. "202505210330.p0.g8f1c8b5.assembly.stream.el9").
+
+    Return Value(s):
+        bool: True if image tag exists on registry.redhat.io, False otherwise.
+    """
+    tag = f"v{version}-{release}"
+    pullspec = f"{DELIVERY_IMAGE_REGISTRY}/{delivery_repo}:{tag}"
+    try:
+        rc, _, stderr = await cmd_gather_async(
```

Ruff (0.15.14)

[warning] Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)